samples/snippets/fsharp/azure/queue-storage.fsx

```diff
@@ -12,102 +11,88 @@ let storageConnString = "..." // fill this in from your storage account
 let storageConnString = 
     CloudConfigurationManager.GetSetting("StorageConnectionString")
 *)
-//
-// Parse the connection string.
-//
-
-// Parse the connection string and return a reference to the storage account.
-let storageAccount = CloudStorageAccount.Parse(storageConnString)
 
 //
 // Create the Queue Service client.
 //
 
-let queueClient = storageAccount.CreateCloudQueueClient()
-
-//
-// Create a queue.
-//
-
-// Retrieve a reference to a container.
-let queue = queueClient.GetQueueReference("myqueue")
-
-// Create the queue if it doesn't already exist
-queue.CreateIfNotExists()
+let queueClient = new QueueClient(storageConnString, "myqueue");
+queueClient.CreateIfNotExists()
 
 //
 // Insert a message into a queue.
 //
 
 // Create a message and add it to the queue.
-let message = new CloudQueueMessage("Hello, World")
-queue.AddMessage(message)
+let message = "Hello, World"
+queueClient.SendMessage(message)
 
 //
 // Peek at the next message.
 //
 
 // Peek at the next message.
-let peekedMessage = queue.PeekMessage()
-let msgAsString = peekedMessage.AsString
+let peekedMessage = queueClient.PeekMessage()
+let msgToString = peekedMessage.ToString()
 
 //
 // Get the next message.
 //
 
 // Get the next message. Successful processing must be indicated via DeleteMessage later.
-let retrieved = queue.GetMessage()
+let retrieved = queueClient.ReceiveMessage()
 
 //
 // Change the contents of a retrieved message.
 //
 
 // Update the message contents and set a new timeout.
-retrieved.SetMessageContent("Updated contents.")
-queue.UpdateMessage(retrieved,
-    TimeSpan.FromSeconds(60.0),
-    MessageUpdateFields.Content ||| MessageUpdateFields.Visibility)
+queueClient.UpdateMessage(
```

Comment:
If the call on L43 fails, what happens when someone tries to reference the `Value` property?   If seems like we should discuss and cover it as a failure scenario.